frontend/src/components/sidebar/PaperMenuItem.tsx

```diff
@@ -21,7 +21,8 @@ export default class PaperMenuItem extends Component<IProps> {
             <Highlighter key="subtitle" search={this.props.searchText}>
                 {
                   this.props.paper.year +
-                  ((this.props.paper.authors) ? ' - ' + this.props.paper.authors.join(', ') : '')
+                  ((this.props.paper.authors) ? ' - ' + this.props.paper.authors.join(', ') : '') + '; ' +
```

Comment:
Interestingly, the backend doesn't provide a list of authors for a paper when searching. But that's totally okay :D 
Haha, I have to say that because I implemented searching :sweat_smile: 
But IF we decide to do this later on, the frontend should just display them as intended.